apps/judicial-system/web/src/utils/hooks/useCase/useCase.logic.ts

```
@@ -0,0 +1,107 @@
+export const formatUpdates = (updates: UpdateCase[], workingCase: Case) => {
+  const changes: UpdateCase[] = updates.map((entry) => {
+    if (entry.force) {
+      return overwrite(entry)
+    }
+
+    return update(entry, workingCase)
+  })
+
+  const newWorkingCase = changes.reduce<UpdateCase>(
+    (currentUpdates, nextUpdates) => {
+      return { ...currentUpdates, ...nextUpdates }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)